app/views/password_resets/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".title" %></h1>
+<div class="row">
+  <div class="mx-auto col-md-6 col-md-offset-3">
+    <%= form_with scope: :password_reset, url: password_resets_path, local: true do |f| %>
```

Comment:
form_with khác gì form_for?

form_for và form_tag là các cách định nghĩa form trước đây của Rails, form_for cho phép tạo form với 1 model đã được xác định (VD model User trong form Sign up), form_tag thường được dùng khi form ko đi kèm với 1 model nhất định mà là gửi request đến 1 Url (VD gửi request đăng nhập trong form Log in)

form_with là cách định nghĩa form mới trong Rails, là sự kết hợp của form_for và form_tag, form_with có thể định nghĩa 1 form kèm với model (VD model: @user) hoặc 1 form ko cần model (VD scope: :session)